HWIOAuthBundle.php

```diff
@@ -55,11 +53,7 @@ public function build(ContainerBuilder $container)
     public function getContainerExtension()
     {
         // return the right extension instead of "auto-registering" it. Now the
-        // alias can be hwi_oauth instead of hwi_o_auth..
-        if (null === $this->extension) {
-            return new HWIOAuthExtension();
-        }
-
-        return $this->extension;
+        // alias can be hwi_oauth instead of hwi_o_auth.
+        return $this->extension ?: new HWIOAuthExtension();
```

Comment:
This looks like old code, I would say the whole method can be removed.

Reply:
That's still case in fact, mainly as explained in comment. After removal Symfony throws exception:
```
LogicException : Users will expect the alias of the default extension of a bundle to be the underscored version of the bundle name ("hwio_auth"). You can override "Bundle::getContainerExtension()" if you want to use "hwi_oauth" or another alias.

Reply:
Ah ok, it's because of the name 🤦  

Maybe we can override [`createContainerExtension`](https://github.com/symfony/symfony/blob/2cee75bef9f6c1ca4e532986dc0dcae8461de4dd/src/Symfony/Component/HttpKernel/Bundle/Bundle.php#L150-L153) instead of [`getContainerExtension`](https://github.com/symfony/symfony/blob/2cee75bef9f6c1ca4e532986dc0dcae8461de4dd/src/Symfony/Component/HttpKernel/Bundle/Bundle.php#L65-L90)? that way we don't have to deal with `$this->extension`.

Reply:
IMHO it's not worth it, and for sure not part of this PR. Feel free to play with it if you want & create the PR 😃

Reply:
no worries, I just saw it because that method was modified.